docs/disclose-cross-border-arrangements-manual-frontend-auto-complete.js

```diff
@@ -0,0 +1,353 @@
+// ==UserScript==
```

Comment:
remove this file - put it in the gitignore file so it doesnt get added accidentally